process/graduation_criteria.md

```diff
@@ -14,6 +14,7 @@ To be accepted in the sandbox a project must
 * Adopt the CNCF [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
 * Adhere to CNCF [IP Policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy) (including trademark transferred)
 * List their sandbox status prominently on website/readme
+* Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md or MAINTAINERS.md file showing the current and emeritus committers. See https://contribute.cncf.io/maintainers/governance/ for more info.
```

Comment:
I'd add "preferably machine parsable markdown", but it does not need to be here in this file, we can add that somewhere under https://contribute.cncf.io/maintainers/governance/

Reply:
We have these terminologies: committer, maintainer, owner, etc.  If there is no other place that clearly defines them, it is better to define them somewhere (could be in another place to which this file can reference)

Reply:
@cathyhongzhang seems like our FAQs could use an update to include those but we should keep that separate for now as we've not had them articulated before and would want them to align with the language used on contribute.cncf.io and other LF materials.

Reply:
@TheFoxAtWork Agree. We need to articulate them clearly and be consistent across CNCF

Reply:
I can reach out to TAG Contributor Strategy to see what they already have and figure out updates to our FAQs outside of this PR.  CC @jberkus @CathPag @geekygirldawn

Reply:
We had a conversation around this with @TheFoxAtWork 

We (TAG Contributor Strategy) have a task for creating a less-technical glossary: https://github.com/cncf/tag-contributor-strategy/issues/140

Reply:
(On the topic of terminology, please have someone look at https://github.com/cncf/foundation/issues/433.)

Reply:
It makes sense for CNCF as a body to have a consistent understanding of what is meant by these different terms, but I don't think it makes sense to impose specific terms on incoming projects, given that no particular governance model is required. An established, successful project shouldn't have make-work imposed on them to rename any roles and terms they may already have in their governance, if that governance is working successfully for the project

Reply:
There are two issues here: 
1) defining the roles so they make sense in the context of governance as a criteria for moving levels (note this specific line is about *sandbox projects* and would impact the TOC's ability to accept them if this is NOT met) and
2) the roles that can act on and receive CNCF benefits (voting, services, etc.)

Given the TOC principles [allow a project to self-govern](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#projects-are-self-governing), the specifics of HOW they choose to define a project governance and committer/maintainer/SteeringCommittee/other process is up to them to do, so long as it exists. Given this specific line is around sandbox, I believe the 'should' statement is acceptable. Projects that apply for sandbox are well positioned if they have more than two or three maintainers, but more-often-than-not are not thinking about their longevity or success until something happens that requires it (thus this being introduced early to think about and consider).

From the principles: 
> governance model that includes a **contribution-based process by which contributors can become committers or maintainers**, and a **clear definition of the top-level project leadership**, with which the foundation will engage and inform and from which it may receive requests for funding and support. 

Separating this discussion out: 
1) Is the language within the changes proposed sufficient to move forward with the expectation that it will change with better definition and expectation of those terms (to be done [outside of this PR](https://github.com/cncf/tag-contributor-strategy/issues/140))?  or does the TAG CS issue need completed first?

2) recommend this be moved to [Craig's cited issue](https://github.com/cncf/foundation/issues/433) 
Those terms carry weight in how the project receives CNCF benefits and as Liz states we shouldn't impose additional work to restructure project governance to conform with that - it goes against the principle I mentioned. If a project has its own roles defined, the project just needs to state to the CNCF of those roles which can act on those benefits (top-level project leadership). 

For myself (reviewing sandbox applications, due diligence, or other project status and health activities) - so long as I can find their roles and understand them, I can align that with the governance needs for moving levels and their ability to exercise those responsibilities in the operation of the project. If I can't get that alignment from what is written, I seek clarity with the project until that alignment is readily conveyed within the governance that is documented.

Reply:
Can we please stop using the word "committer"?  It is not immediately comprehensible to modern projects, and it's different from the word we use for project leadership elsewhere, which is "maintainer".

Reply:
> it's different from the word we use for project leadership elsewhere, which is "maintainer".

Not all committers are project leadership, especially in larger projects.  (See [Craig's cited issue](https://github.com/cncf/foundation/issues/433).)

Is the intention of this criteria to only refer to project leadership?  I read it as talking about other kinds of project membership too.

Reply:
Commiter is the term used in the foundation charter. A small preamble in the doc like 'For the purposes of the TOC docs and process, a maintainer is equivalent to a committer as defined by the CNCF charter' or something like that.

Reply:
Additionally, I had the impression that this change WAS intended only to define how project leadership is selected.  @TheFoxAtWork, can you clarify?

Reply:
Could I kindly ask that instead of the TOC defining apples to mean oranges, that any change is holistic and updates the term used in the CNCF charter as well?

Reply:
> Again, few current github-based projects use the term "committer",

Cilium is an example of a project that uses this term. And like @mrbobbytables said, it's in the charter. 

Before going to the effort of changing the charter, maybe step back and see what problem this is trying to solve? Given that we don't impose any specific governance, and projects likely come into the organization with some pre-defined governance docs in place, forcing everyone into the same terminology is just make-work. It doesn't **help projects**.

Reply:
TAG CS now has the first version of the non-technical glossary: https://contribute.cncf.io/resources/glossary/#project-maintainer.

Reply:
> In CNCF, term “maintainer” is used over other similar terms such as “committer”.

But not necessarily in CNCF _projects_; I don't think it is the intention of the TOC to make projects use a standard noun, or if it is to be part of the 'light' oversight that projects are subject to, that it be very clearly communicated.

Reply:
Thanks for the feedback @craigbox 
I would be very happy to update the non-technical glossary with the outcome of this discussion (or a separate discussion).

Reply:
Raised an issue against the [Glossary](https://github.com/cncf/tag-contributor-strategy/issues/471) about this.

Reply:
I will echo what @craigbox says here: when projects sign up to join the CNCF they are assured that they won't be required to adopt a standard governance model. As a consequence, they have no expectation that they will be required to adopt some standardized language within their existing governance documentation